### PR TITLE
Fix JWT claims request

### PR DESCRIPTION
**Pull request by linoman**
**What is this feature?**

Either username or email claims are required, not both.

### CODE REVIEW

**Comment by mgyongyosi** [comment]
@linoman Could you please add 1-2 unit test(s) to verify the functionality?

**Review by IevaVasiljeva — APPROVED** [review]

**Review by Jguer — APPROVED** [review]

File: pkg/services/authn/clients/jwt.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func (s *JWT) Authenticate(ctx context.Context, r *authn.Request) (*authn.Identi
 		}
 	}
 
-	if id.Login == "" || id.Email == "" {
+	if id.Login == "" && id.Email == "" {
 		s.log.Debug("Failed to get an authentication claim from JWT",
 			"login", id.Login, "email", id.Email)
 		return nil, ErrJWTMissingClaim.Errorf("missing login or email claim in JWT")
```
Comment (IevaVasiljeva) [inline pkg/services/authn/clients/jwt.go:116]: nit: should correct this message as well:
```suggestion
		return nil, ErrJWTMissingClaim.Errorf("missing login and email claim in JWT")
```

Comment (linoman) [reply pkg/services/authn/clients/jwt.go]: Changed as requested.

**Comment by linoman** [comment]
> @linoman Could you please add 1-2 unit test(s) to verify the functionality?

@mgyongyosi I've expanded an existing test covering multiple config scenarios. Please let me know if this is what you had in mind.

**Review by mgyongyosi — APPROVED** [review]

**Pull request merged**